server/initializers/migrations/0295-add-originally-published-at.ts

```diff
@@ -0,0 +1,42 @@
+import * as Sequelize from 'sequelize'
```

Comment:
It's the migration number 325 now, and you need to update the LAST_MIGRATION_VERSION: https://github.com/Chocobozzz/PeerTube/blob/develop/server/initializers/constants.ts#L19